source/team-guide/accessing-aws-accounts.html.md.erb

```diff
@@ -0,0 +1,53 @@
+---
+title: Accessing AWS accounts
+last_reviewed_on: 2022-01-10
+review_in: 6 months
+---
+
+# <%= current_page.data.title %>
+
+## How we access AWS accounts
+
+We have two common methods for accessing AWS accounts:
+
+- Through a [common portal](https://moj.awsapps.com/start#/).
+- Through a [superuser account](https://user-guide.modernisation-platform.service.justice.gov.uk/team-guide/adding-a-new-team-member.html#adding-them-as-a-superadmin-in-our-aws-landing-zone).
+
+### Common portal access
+
+Using a web browser, an authenticated user can navigate to the [common portal](https://moj.awsapps.com/start#/) and select
+an AWS account. The portal will allow them to log in to the AWS console with read-only privileges, or to retrieve an
```

Comment:
```suggestion
an AWS account. The portal will allow them to log in to the AWS console with the relevant privileges, or to retrieve an
```
We have different levels, admin/developer/readonly/etc depending on the persons role and the account